src/sage/schemes/hyperelliptic_curves/monsky_washnitzer.py

```diff
@@ -513,12 +522,12 @@ def __init__(self, Q, laurent_series=False):
             raise ArithmeticError("2 and 3 must be invertible in the "
                                   "coefficient ring (=%s) of Q" % base_ring)
 
-        # CommutativeAlgebra.__init__ tries to establish a coercion
+        # .__init__ tries to establish a coercion
         # from the base ring, by github issue #9138. The corresponding
         # hom set is cached.  In order to use self as cache key, its
         # string representation is used. In otder to get the string
         # representation, we need to know the attributes _a and
-        # _b. Hence, in #9138, we have to move CommutativeAlgebra.__init__
+        # _b. Hence, in #9138, we have to move .__init__
```

Comment:
This is somewhat of a strange comment. It would be good to fix this here by just making this class a `UniqueRepresentation`. If you want to leave the comment for now, it would be better to say `Parent.__init__` here.